script.js

```diff
@@ -0,0 +1,635 @@
+const container = document.createElement("div");
+const textarea = document.createElement("textarea");
+const keyID = [
```

Comment:
first of all - it is constant - has to be in uppercase.
secondly - such constants have to be stored in a separate file and be exported here.

Reply:
P.S.  keyID can be generated from keysEn

Reply:
changed const to uppercase